src/risedevtool/etcd.toml

```diff
@@ -26,7 +26,7 @@ if [ "$ETCD_SYSTEM" = "darwin-amd64" ] || [ "$ETCD_SYSTEM" = "darwin-arm64" ] ||
     unzip "${ETCD_DOWNLOAD_PATH_OTHER}" -d "${PREFIX_TMP}"
 else
     curl -L "${ETCD_DOWNLOAD_URL_LINUX}" -o "${ETCD_DOWNLOAD_PATH_LINUX}"
-    tar -xf "${ETCD_DOWNLOAD_PATH_LINUX}" -C "${PREFIX_TMP}"
+    tar -xf "${ETCD_DOWNLOAD_PATH_LINUX}" -C "${PREFIX_TMP}" --no-same-owner
```

Comment:
According to man tar, `no-same-owner` seems to be the default config. Why change this?

Reply:
```
     --no-same-owner
             (x mode only) Do not extract owner and group IDs.  This is the reverse of --same-owner and the default behavior
             if tar is run as non-root.
```

Reply:
Okay, I see:

> src/risedevtool/etcd.toml: because the tar command here may report an error when executed in a container, I add the --no-same-owner parameter to avoid this problem.

We can specify running user in docker. Changing this will cause problems for developers running risedev on their own machine. So don't get this changed.